visualizations/dynkin-diagram-game/static/app.js

```
@@ -35,7 +35,7 @@ const MESSAGES = {
     eyebrow: "インタラクティブ・リー理論", title: "ディンキン図形ビルダー",
     lede: "ノードを1つずつ追加して図形を構成します。エディターは全階数8までの有限結晶型を判定し、対応するカルタン行列を表示します。",
     mode: "モード", move: "移動", addNode: "ノードを追加", connect: "接続", bond: "辺",
-    single: "単辺", double: "二重辺", triple: "三重辺", reverse: "矢印を反転", delete: "削除",
+    single: "単辺", double: "二重辺", triple: "三重線", reverse: "矢印を反転", delete: "削除",
```

Biome (2.5.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)